### PR TITLE
Add Python 3.6, drop unsupported 2.6, 3.2, 3.3

### PR DESCRIPTION
**Pull request by hugovk**
## Reasons for dropping old ones

* https://en.wikipedia.org/wiki/CPython#Version_history

### 2.6

* https://snarky.ca/stop-using-python-2-6/
* http://www.curiousefficiency.org/posts/2015/04/stop-supporting-python26.html
* Current pip 9 deprecates Python 2.6 support, pip 10 won't support it (https://github.com/pypa/pip/issues/3955)

### 3.2

* pip 8 no longer supports Python 3.2 ([link](https://pip.pypa.io/en/stable/news/))
* Coverage 4 no longer supports Python 3.2 ([link](https://bitbucket.org/ned/coveragepy/issues/407/coverage-failing-on-python-325-using))
* Requests no longer supports Python 3.2

### 3.3

* pip 10 will deprecate Python 3.3 support, pip 11 won't support it (https://github.com/pypa/pip/issues/3796)

### CODE REVIEW

**Comment by RouxRC** [comment]
For once I'm not completely in favor of this: dropping 2.6 or 3.2 can potentially break existing deployed software on machines still relying on those (full disclosure, I run bots using python-twitter on old debian servers under python2.6 ;( )
Although I understand it is now officially deprecated, so I can understand endorsing it now makes sense, and bad server maintainers like myself should then probably just freeze their pip requirements on packages depending on this.
So my apologies for this time @hugovk, I'll respectfully let @sixohsix decide on this one ;)

**Comment by hugovk** [comment]
Here's how many pip installs Python Twitter Tools has had from PyPI over the past month:

```
$ pypinfo --pip --percent twitter pyversion
python_version percent download_count
-------------- ------- --------------
2.7              79.2%         17,371
3.6              10.0%          2,205
3.5               8.9%          1,945
3.4               1.6%            357
3.3               0.1%             29
3.7               0.1%             22
2.6               0.0%              9
3.2               0.0%              5
```

**Comment by sixohsix** [comment]
I'm with RouxRC on this. The arguments are not convincing to me.

Though users running 3.5 or 2.6 will probably experience pain trying to work with updated versions of pip, coverage, requests, etc. there is no reason to believe that they will encounter any problems with this twitter package. The code hasn't introduced any breaking changes so far as I know.

I would merge this change when either of two things happen:

- we introduce code changes that we know will break with 2.6 or 3.5.
- users encounter problems running this package with 2.6 or 3.5 (that is: when this happens, rather than put in the effort to fix this package, we might opt to deprecate those versions instead.)

In short, I do not want to merge this at this time.
